packages/debug/src/browser/debug-tab-bar-decorator.ts

```diff
@@ -0,0 +1,59 @@
+/********************************************************************************
+ * Copyright (C) 2020 Ericsson and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import { injectable, inject } from 'inversify';
+import { DebugSessionManager } from './debug-session-manager';
+import { DebugWidget } from './view/debug-widget';
+import { Emitter, Event } from '@theia/core/lib/common/event';
+import { TabBarDecorator } from '@theia/core/lib/browser/shell/tab-bar-decorator';
+import { FrontendApplication, FrontendApplicationContribution, Title, Widget } from '@theia/core/lib/browser';
+import { WidgetDecoration } from '@theia/core/lib/browser/widget-decoration';
+import { DisposableCollection } from '@theia/core/lib/common';
+
+export const DebugSessionManagerGetter = Symbol.for('DebugSessionManagerGetter');
```

Comment:
We don't use `Symbol.for`.

Although I don't like an idea with `DebugSessionManagerGetter` generally. What was the cycle?